prediction_market_agent/agents/microchain_agent/microchain_agent.py

```diff
@@ -53,6 +91,11 @@ def build_agent_functions(
         functions.append(
             RememberPastActions(long_term_memory=long_term_memory, model=model)
         )
+
+    if build_dynamic_contract_functions:
```

Comment:
I did suggest this, but I also said I didn't like the idea 😀. IMO there's not much to gain from deploying an agent with just this boolean flag switched to on. Or are you curious to see how it behaves?

Reply:
I removed this, agree with your points.